README.md

```diff
@@ -1,138 +1,147 @@
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [🎥 Walkthrough Video](#loom-walkthrough)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Ruby-Group-Capstone-Project] <a name="about-project"></a>
+
+The "Ruby-Group-Capstone-Project" project is a Ruby exercise designed to help users warm up and work as a group.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- FEATURES -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add classes**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- WALKTHOROUGH VIDEO -->
+
+## 🎥 Loom Walkthrough <a name="loom-walkthrough"></a>
+
+- [Walkthrough video Link](https://drive.google.com/uc?id=1x0gkUq6E7lVjyE6K3ocFNEw9z0smPy8x&)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:everinurmind/Ruby-Group-Capstone-Project.git
+```
+
+### Usage
+
+To run the project, execute the following commands:
+
+```sh
+  ruby main.rb
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Nurbol Sultanov**
+
+- GitHub: [@everinurmind](https://github.com/everinurmind)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/everinurmind/)
+
+👤 **Ghulam Reza Rajabi**
+
+- GitHub: [@ghreza-crypto](https://github.com/ghreza-crypto)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ghulam-reza-rajabi-7a9aa3142/?originalSubdomain=af)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **[Decorating a class]**
+- **[Setting up associations]**
+- **[Adding basic UI]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/everinurmind/Ruby-Group-Capstone-Project/tree/issues).

```

Comment:
- [optional] please, your issue page link is broken. Kindly add a working one for a professional README file.